web/locales/en/messages.ftl

```diff
@@ -642,8 +642,8 @@ contribute-more =
        *[other] Ready to do { $count } more?
     }
 record-cta = Start recording
-record-platform-not-supported = We’re sorry, but your platform is not currently supported.
-record-platform-not-supported-desktop = On desktop computers, you can download the latest:
+record-platform-not-supported = We’re sorry, your platform is not currently supported.
+record-platform-not-supported-desktop = On desktop devices you can contribute by downloading...
```

Comment:
I decided not to create new keys for these two specifically because the difference in semantics didn't seem big enough to merit retranslation in all the languages, but I'm not married to that opinion.

Reply:
All good, makes sense to me!